Dockerfile

```diff
@@ -13,6 +13,7 @@ WORKDIR /app
 #   1. Create directories explicitly because docker use the wrong permission for explicit creation.
 #   2. For the rest, set the default ACL to 777 for all users.
 RUN mkdir -m 777 /app/Megatron-LM /app/examples /app/fast_llm /app/tests /app/tools \
+    && install -m 777 /dev/null /app/fast_llm/__init__.py \
```

Comment:
How about we copy with the other setup files instead? We'll want it to recover the version anyway. 
Also why /dev/null?
Do you know what changed that broke the install?

Reply:
Ok thanks for the clarification. Let's replace the file creation with copy, then we can merge.